packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -798,21 +802,21 @@ class DebugTimingLogger {
   DebugTimingLogger(this.name, {this.mute});
 
   final String name;
-  final bool mute;
+  final bool? mute;
```

Comment:
instead of making this nullable, just make this.mute required

Reply:
I've set it to false by default.